### ISSUE TITLE
Loading topfind doesn't work

### ISSUE THREAD

**Issue by murmour** [protz/ocaml-installer#53]
In the interactive toplevel (`ocaml`), loading `topfind` fails:

```
$ ocaml
        OCaml version 4.02.3

# #use "topfind";;
Cannot find file topfind.
```

The `topfind` file does exist:
```
$ test -f /home/mrm/.opam/system/lib/toplevel/topfind ; echo $?
0
```

And the environment variable is set correctly:
```
$ set | grep OCAML_TOPLEVEL_PATH
OCAML_TOPLEVEL_PATH=/home/mrm/.opam/system/lib/toplevel
```

If you put the file into current directory and name it `topfind.ml`, then `#use "topfind"` works.

And no, just renaming `topfind` into `topfind.ml` (without copying) doesn't work. It's probably that `ocaml` doesn't use the environment variable correctly, due to some path-related bug.

**Comment by dbuenzli** [comment]
I don't remember who's responsible for `OCAML_TOPLEVEL_PATH` and what it's supposed to do but it's it's not documented in the ocaml manual and a `git grep` on the ocaml repo returns nothing. 

The issue here is that without additional `-I` given to `ocaml` a `#use` will only lookup in the `cwd` and `ocamlc -where`. So `topfind` should be installed in the latter directory, could you check the result of: 

```
test -f $(ocamlc -where)/topfind ; echo $?
```

**Comment by murmour** [comment]
Indeed, that's the problem:
```
$ test -f $(ocamlc -where)/topfind ; echo $?
1

$ ocamlc -where
C:\OCaml\lib
```

I've tried copying `topfind` into `C:\OCaml\lib\toplevel\`, and it didn't help.
Possible solution: `ocamlc -where` should return `/cygdrive/c/OCaml/lib`.

**Comment by dbuenzli** [comment]
> I've tried copying topfind into C:\OCaml\lib\toplevel\

You should copy it to `C:\OCaml\lib`.

**Comment by murmour** [comment]
>> I've tried copying topfind into `C:\OCaml\lib\toplevel\`

> You should copy it to `C:\OCaml\lib`.

I did so, and it works! Thank you very much for assistance.
The installer should put it there as well -- it's how the issue is to be solved.

**Comment by protz** [comment]
Just for the record I'm eagerly waiting on David Allsopp's rumored, windows-compatible version of OPAM 2.0-beta. Once that is out, I'm hoping to decommission the installer.

**Comment by murmour** [comment]
Yeah, the rumors of @dra27's miracles are getting really strong lately. But for now, Jonathan, you are the champion of the forsaken souls on Windows. We worship you!

(starts drawing the pentagram)

**Comment by protz** [comment]
Hahah yes, indeed! Actually, https://github.com/fdopen/opam-repository-mingw works pretty well, and Andreas just recently released his patches
